src/components/map/MapService.js

```diff
@@ -2073,10 +2073,26 @@ goog.require('ga_urlutils_service');
         var mustReorder = false;
 
         var addTopicSelectedLayers = function() {
-          addLayers(gaTopic.get().selectedLayers.slice(0).reverse());
-          var activatedLayers = gaTopic.get().activatedLayers;
-          if (activatedLayers.length) {
-            addLayers(activatedLayers.slice(0).reverse(), null, false);
+          // if plConf is active, we get layers from there. This
+          // might include opacity and visibility
+          var topic = gaTopic.get();
+          if (topic.plConfig && topic.plConfig.length) {
+            var p = gaUrlUtils.parseKeyValue(topic.plConfig);
+            addLayers(p.layers ? p.layers.split(',') : [],
```

Comment:
Do we have cases where `layers` parameter is defined but layers_opacity, layers_visibility and layers_timestamp are not?